api/HTMLShadowElement.json

```diff
@@ -0,0 +1,114 @@
+{
+  "api": {
+    "HTMLShadowElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLShadowElement",
+        "support": {
+          "webview_android": {
+            "version_added": "37"
```

Comment:
This should be `"35"`.

Reply:
MDN states that the minimum supported version for Android is `37` however.

Reply:
MDN is wrong. I work for google. I verify these things by looking them up in commit history or talking to the engineers who built them.